### ISSUE TITLE
Testing Resolutions

### ISSUE THREAD

**Issue by SalOMac** [rpgboss/rpgboss#129]
I tried it out and while the in game resolution (actually wouldn't view widow be more accurate?) While it does change, it doesn't account for resources made for the new resolution. For instance I tried to implement a title  screen of 1024x576 for the resolution of that size and it still looks like this: 

![resotest](https://cloud.githubusercontent.com/assets/9708508/5022032/c84e6da0-6aa3-11e4-8270-ff7b2f65ac5c.png)

Basically it's been squished in to fit a 3:4 resolution instead of the 9:16 I chose. Just thought I'd pass it by you.
